db.go

```diff
@@ -45,6 +45,7 @@ import (
 var DefaultOptions = &Options{
 	WALFlushInterval:  5 * time.Second,
 	RetentionDuration: 15 * 24 * 60 * 60 * 1000, // 15 days in milliseconds
+	MaxBytes:          0, // Don't use size based retention by default, uses 0 instead of -1 in case it is unintialized
```

Comment:
This seems confusing. (I for one don't understand what the comment is saying.)

Why not have both this and the flag default to zero?

Reply:
I suppose it wasn't very clear, when running it through the tests it failed because I was assuming the default would be -1, indicating that I don't want to limit the data storage by size. But one of the test supplied an options of 

```
// reopen DB from snapshot
	db, err = Open(snap, nil, nil, &Options{
		RetentionDuration: 10,
		BlockRanges:       []int64{50},
	})
```
, which didn't include the MaxBytes field, and so MaxBytes was assigned 0 instead of -1. So it was attempting to limit the storage size to 0. I changed it to 0 now as the default, to indicate no limit, in the case that MaxBytes isn't supplied in the options when opening the database. I can't really think of a reasonable use case for 0 as the MaxBytes so I don't think this will be an issue. The flag didn't get changed to 0 too simply because i forgot to do it, I'll make that adjustment now.

Reply:
Thanks, it's clearer now.

You should probably also drop the -1 from the comment too, to avoid any confusion.